README.md

```diff
@@ -28,27 +28,27 @@ If you'd like to discuss the tool, development, or contributions, we are in the
 
 ## Developing
 
-There is some documentation in the [docs/](./docs/) directory, notably [writing an intent](./docs/writing-an-intent.md).
+Documentation on the various parts and aspects of firefox-voice could be found in the [docs/](./docs/) directory, notably [writing an intent](./docs/writing-an-intent.md). 
 
-Note:
+However to quickly get started with the development environment, you would need to:
 
-1. Download the latest LTS version of NodeJs before running above npm commands.
-2. After downloading Firefox Nightly, change its name to FirefoxNightly (no space), otherwise the project wont be able to spawn the firefox.
+1. Download the latest LTS version of [NodeJs](https://nodejs.org/) before running any of the provided npm commands.
+2. Download [Firefox Nightly](https://www.mozilla.org/en-US/firefox/channel/desktop/) which is a pre-release version of Firefox that is updated every now and then just so you are developing and testing with the latest firefox features.
+3. Use `tar xjvf` to extract the binaries for FireFox Nightly into a location of your choice could be `/opt/FireFoxNightly/` or just your download folder, wherever is fine. 
+4. Let firefox-voice know where you kept the binary for FireFox Nightly typically by runing `export FIREFOX=/path/to/FireFoxNightly/firefox` and using our example location would be `export FIREFOX=/opt/FireFoxNightly/`
 
-The developer installation is:
+You would have to run this everytime you have a new session so may want to look [how to let the export stick permanently](https://stackoverflow.com/questions/13046624/how-to-permanently-export-a-variable-in-linux)
+
+Finally, install dependencies and start firefox voice by running:
 
 ```sh
 npm install
 npm start
 ```
 
-This will launch a new Firefox browser with the extension installed. You should probably have [Nightly or Developer Edition](https://www.mozilla.org/en-US/firefox/channel/desktop/) installed.
```

Comment:
Please, leave this line intact so the user knows what to expect after running the commands above